website_and_docs/content/documentation/grid/getting_started.en.md

```diff
@@ -16,8 +16,8 @@ aliases: [
     * Java 11 or higher installed
     * Browser(s) installed
     * Browser driver(s)
-      * If using Selenium 4.6, Selenium Manager will configure the drivers for Chrome, Firefox, and Edge [if they are not found on the `PATH`]({{< ref "../webdriver/getting_started/install_drivers.md#1-selenium-manager-beta" >}}).
-      * [Installed and on the `PATH`]({{< ref "../webdriver/getting_started/install_drivers.md#3-the-path-environment-variable" >}})
+      * If using Selenium 4.6+, Selenium Manager will configure the drivers automatically if you add `--selenium-manager true`.
```

Comment:
Should we add a link to the `Selenium Manager` document that is available in this PR for cross reference purposes?

Reply:
I wasn't going to touch the grid page except I saw it was outdated. Whatever makes sense.

Reply:
ok, yeah, I'll add the link. Thanks.